src/ct/ct_pref_dlg_text_n_code.cc

```diff
@@ -99,6 +99,7 @@ Gtk::Widget* CtPrefDlg::build_tab_text_n_code()
     Gtk::Entry* entry_timestamp_format = Gtk::manage(new Gtk::Entry());
     entry_timestamp_format->set_text(pConfig->timestampFormat);
     Gtk::Button* button_strftime_help = Gtk::manage(new Gtk::Button());
+    button_strftime_help->set_tooltip_text(_("Online _Manual"));    
```

Comment:
This won't be accepted because has an underscore accelerator but it's not a menu

Reply:
Hi, Giuseppe. I realized that the " _ " sign is used to indicate the menu. But then you will need to enter additional lines to translate the names of pop-ups and tooltips. Do you agree?"

Reply:
For example "Online Manual"